app-shells/starship/metadata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>randy@electronsweatshop.com</email>
+		<name>Randy Barlow</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription lang="en">
+		Starship is:
+
+		Fast: it's fast – really really fast!
+		Customizable: configure every aspect of your prompt.
+		Universal: works on any shell, on any operating system.
+		Intelligent: shows relevant information at a glance.
+		Feature rich: support for all your favorite tools.
+		Easy: quick to install – start using it in minutes.
+	</longdescription>
+	<upstream>
+		<remote-id type="github">starship/starship</remote-id>
+	</upstream>
+	<use>
+		<flag name='battery'>Include support for laptop battery information.</flag>
+		<flag name='http'>If enabled, starship can help you file bugs with itself</flag>
+		<flag name='tls-vendored'>Use a bundled openssl instead of the system openssl</flag>
```

Comment:
It's more common to use a system-* flags (e.g. `system-openssl`) enabled by default.

... but personally I'd just unconditionally use system's openssl and not provide a flag though, especially for openssl (wouldn't want to have to worry about bundled openssl vulnerabilities).

Reply:
I dropped all use flags, so we use the system openssl unconditionally now.